src/utils.ts

```diff
@@ -61,8 +68,8 @@ const toMetricsHandler = (log: (item: string) => void, report: Reporter) => (
   log(`(${metricType}) ${baseKey}: ${duration} [${tags}]`)
   log(`(${metricType}) ${keyWithConclusion}: ${duration} [${tags}]`)
 
-  report[metricType](baseKey, duration, tags as string[])
-  report[metricType](keyWithConclusion, duration, tags as string[])
```

Comment:
Really don't know why I was doing `as const` before since it wasn't getting us anything.